test/itkElastixRegistrationMethodTest.cxx

```diff
@@ -77,16 +77,26 @@ int itkElastixRegistrationMethodTest( int argc, char * argv[] )
   using ReaderType = itk::ImageFileReader< ImageType >;
   ReaderType::Pointer fixedImageReader = ReaderType::New();
   fixedImageReader->SetFileName( fixedImageFileName );
+  fixedImageReader->Update();
 
   using ReaderType = itk::ImageFileReader< ImageType >;
   ReaderType::Pointer movingImageReader = ReaderType::New();
   movingImageReader->SetFileName( movingImageFileName );
+  movingImageReader->Update();
 
   ShowProgress::Pointer showProgress = ShowProgress::New();
   filter->AddObserver( itk::ProgressEvent(), showProgress );
   filter->SetFixedImage( fixedImageReader->GetOutput() );
   filter->SetMovingImage( movingImageReader->GetOutput() );
 
+  //typename FilterType::ParameterObjectPointer parameterObject = filter->GetParameterObject();
```

Comment:
Maybe pass an additional command line parameter, and enable this (currently commented-out code) if it is non-zero?

Reply:
These are set in the constructor -- I'll remove these.